doc/source/api/stats.rst

```diff
@@ -17,3 +17,4 @@ Stats
     r2_score
     summary
     waic
+    kde
```

Comment:
can you move it so everything keeps being alphabetically sorted

Reply:
Sure